pkgs/applications/version-management/git/default.nix

```
@@ -303,6 +303,7 @@ stdenv.mkDerivation (finalAttrs: {
       mkdir -p $out/share/git
       cp -a contrib $out/share/git/
       mkdir -p $out/share/bash-completion/completions
+      rm -f $out/share/bash-completion/completions/git
```

```suggestion
      rm $out/share/bash-completion/completions/git
```
Don't we want to notice if the file vanishes?